content/en/docs/Tasks/creating-an-update-graph.md

```diff
@@ -116,6 +120,43 @@ The CSV name currently installed must be in the `skips` field of a CSV in the ne
 
 The version currently installed must be in the `olm.skipRange` field of a CSV in the new channel.
 
+
+`replaces and skips example:`
+
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: etcdoperator.v0.9.2
+  namespace: placeholder
+  annotations:
+spec:
+    displayName: etcd
+    description: Etcd Operator
+    replaces: etcdoperator.v0.9.0
+    skips:
+    - etcdoperator.v0.9.1
+```
+
+The `replaces` field contains the current installed CSV. `Skips` parameter contains the version that needs to skip during the upgrade process. There might be some operator versions that have the bug. A `skips` parameter is used to ignore the faulty upgrade of the operator.
+
+#### Exmple for SkipRange
+
+`skipRange example:`
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+    name: elasticsearch-operator.v4.1.2
+    namespace: placeholder
+    annotations:
+        olm.skipRange: '>=4.1.0 <4.1.2'
+```
+
+The skipRange parameter is used to ignore the multiple versions of an operator during the upgrade cycle. In the above example the operator version will be avoided from 4.1.0 to 4.1.1.
+
 ## Add Modes
 
-TODO
+[Install Modes and Supported Operator Groups](/docs/concepts/crds/operatorgroup/#installmodes-and-supported-operatorgroups)
```

Comment:
Why we need the Add Notes? 
What is the intention to add this link here?

Reply:
Hmm, it looks like this was a previous placeholder for documenting the various skip/replaces/etc. properties? I think we can either punt off updating this section, and keeping the TODO in place, or eliminate (well, or re-organize this PR) this section entirely.

Reply:
TODO is already part of the old version file with title of Add Modes. We already have explanation of Add modes in some another document that's why I just provided the link in TODO.

Reply:
If it is not recommended then we can completely remove TODO from this document.

Reply:
I don't have a super-strong opinion either way - I'd think we'd like to remove as many hanging TODOs as much as possible, but if we want to tackle that outside of this PR, then that's fine with me. That said, just replacing the existing `TODO` comment here with a link to the install modes documentation seems strange given we have a dedicated `Add Modes` top-level header that just links to more documentation.

@anik120 Any preference here? Seems like it would be easier to punt off for another PR.

Reply:
@laxmikantbpandhare My previous comment on this Add Modes section was referring to the fact that you're linking to a section that explains InstallModes instead of Add Modes.

Add Modes define the channel graph's upgrade behavior for when you add a bundle to the index e.g `replaces`, `semver`, `semver-skippatch`. InstallModes defines what namespaces an operator is meant to watch which is unrelated to this doc.

See:
https://github.com/operator-framework/operator-registry/blob/e29651f8ecf77b32fb51491bbc395ebb2e527161/cmd/opm/index/add.go#L65
https://github.com/operator-framework/operator-registry/blob/master/pkg/registry/channelupdateoptions.go#L16-L27

So we'd need something like this doc to explain each of the modes if we want to address this TODO.
https://docs.google.com/document/d/1N29w764eZroOywvLK1tb00XqX40A9yo4ghmSLE9PaMI/edit?usp=sharing

Or if we want to do it as a follow up PR then leave it as a TODO until we can add that in.

Reply:
+1 to what Haseeb said